.gitignore

```diff
@@ -14,4 +14,8 @@
 !var/SymfonyRequirements.php
 /vendor/
 /web/bundles/
-/web/uploads/
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
??

Reply:
resolved in jakowaty/bip.siecobywatelska.pl master